### PR TITLE
[feature1] First feature was implemented

### PR DESCRIPTION
**Pull request by jbodart-argenx**
The following changes were made:

- add file `new-feature.txt`
- create folder `feature-1`
- add file `feature-1/feat-1-file1.txt`

### CODE REVIEW

**Review by jmbo1190 — CHANGES_REQUESTED** [review]
please describe rationale for that change

File: feature-1/feat-1-file1.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+This file (feat-1-file1.txt) was created in local branch feature1.
+
+Add one line in local branch feature1.
```
Comment (jbodart-argenx) [inline feature-1/feat-1-file1.txt:3]: I would rather change this

File: new-feature.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+This file (new-feature.txt) was created locally in feature1 branch.
```
Comment (jbodart-argenx) [inline new-feature.txt:1]: this is excellent (👍 )

File: feature-1/feat-1-file1.txt [hunk 3]
```diff
@@ -0,0 +1,3 @@
+This file (feat-1-file1.txt) was created in local branch feature1.
```
Comment (jbodart-argenx) [inline feature-1/feat-1-file1.txt:1]: good catch! ;-)

**Review by jbodart-argenx — COMMENTED** [review]
good job, see minor comments

File: feature-1/feat-1-file1.txt (hunk 3 above)
Comment (jmbo1190) [reply feature-1/feat-1-file1.txt:1]: thanks

File: feature-1/feat-1-file1.txt (hunk 1 above)
Comment (jmbo1190) [reply feature-1/feat-1-file1.txt:3]: Like this: "This line was added locally in `feature1` branch" ?

**Review by jmbo1190 — APPROVED** [review]

**Comment by jmbo1190** [comment]
Please merge this PR

**Comment by jbodart-argenx** [comment]
merged!

**Pull request merged**